MODULE.bazel

```diff
@@ -29,9 +29,13 @@ maven.install(
     "com.google.errorprone:error_prone_annotations:2.16",
     "com.google.errorprone:error_prone_type_annotations:2.16",
     "com.google.code.findbugs:jsr305:3.0.2",
+    "com.squareup:javapoet:1.12.0",
     "com.google.j2objc:j2objc-annotations:1.3",
     "com.github.stephenc.jcip:jcip-annotations:1.0-1",
     "org.checkerframework:checker-qual:3.12.0",
+    "com.ryanharter.auto.value:auto-value-gson-extension:1.3.1",
+    "com.ryanharter.auto.value:auto-value-gson-runtime:1.3.1",
+    "com.ryanharter.auto.value:auto-value-gson-factory:1.3.1",
   ],
   repositories = [
     "https://dl.google.com/android/maven2",
```

Comment:
shouldn't we uncomment the `lock_file` line?

Reply:
Done

Reply:
remove the TODO too